assignments/prototypes.js

```diff
@@ -10,38 +10,74 @@
   
 /*
   === GameObject ===
-  * createdAt
-  * name
-  * dimensions (These represent the character's size in the video game)
-  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  *11 createdAt
+  * 11name
+  * 11dimensions (These represent the character's size in the video game)
+  * 11destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  * 
+  * DONE
 */
+"use strict"
+function GameObject (attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+}
+
+GameObject.prototype.destroy = function  () {
+  return (`${this.name} was removed from the game`)
+}
 
 /*
   === CharacterStats ===
-  * healthPoints
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
+  11* healthPoints
+  11* takeDamage() // prototype method -> returns the string '<object name> took damage.'
+  11* should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats (attribute) {
+  GameObject.call(this, attribute)
+  this.healthPoints = attribute.healthPoints;
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype)
+
+CharacterStats.prototype.takeDamage = function(){
+  return (`${this.name} took damage`)
```

Comment:
This is making a handy copy of the parents methods and saving it to the childs prototype, giving it access to those same methods as well as its own. (this keeps you from rewriting it 100000 times for each child of the `GameObject`